README.md

```diff
@@ -1,35 +1,80 @@
-# Public BI benchmark
+# Public BI benchmark for TiDB
 
-User generated benchmark derived from the DBTest'18 paper [1] by Tableau. It contains real data and queries from 46 public workbooks in Tableau Public [2].
+## 数据集大小
 
-We downloaded 46 of the biggest workbooks and converted the data to *.csv* files and collected the SQL queries that appear in the Tableau log when the workbooks are visualized. We processed the *.csv* files and queries with the purpose of making them load and run on different database systems.
-
-This repository contains samples and schema of each .csv file, the queries and a set of scripts for working with the benchmark. The full *.csv* files are available in compressed *bzip2* format at: https://www.cwi.nl/~boncz/PublicBIbenchmark/
+There are 46 workbooks containing 206 tables (.csv files) with the total size of 41 GB compressed and 386 GB uncompressed.
 
-Each directory is associated with a workbook and contains:
+## 使用方法
 ```
-samples:        a sample of each .csv file (first 20 rows)
-tables:         .sql files containing the schema of each .csv file
-queries:        .sql files containing the queries
-data-urls.txt:  links for downloading the full .csv.bz2 compressed files
+go run main.go [download|prepare|run|cleanup]: 
+  -P int
+    	MySQL Port (default 3306)
+  -c value
+    	test case you want to benchmark
+  -d string
+    	MySQL database (default "test")
+  -h string
+    	MySQL Host (default "127.0.0.1")
+  -i string
+    	input benchmark resource (default "./benchmark")
+  -ignore-run-error
+    	Ignore errors when run
+  -o string
+    	output directory to save cvs files and logs (default "./var")
+  -p string
+    	MySQL Password
+  -sample
+    	Use sample to test
+  -u string
```

Comment:
How about reorg the options like this:
1. options used to connect the database server.
2. options used to control the test behavior.